packages/contracts-bedrock/snapshots/storageLayout/L1CrossDomainMessenger.json

```diff
@@ -113,17 +106,52 @@
   },
   {
     "bytes": "20",
-    "label": "otherMessenger",
+    "label": "spacer_207_0_20",
     "offset": 0,
     "slot": "207",
-    "type": "contract CrossDomainMessenger"
+    "type": "address"
   },
   {
-    "bytes": "1376",
-    "label": "__gap",
+    "bytes": "20",
+    "label": "xDomainMsgSender",
     "offset": 0,
     "slot": "208",
-    "type": "uint256[43]"
```

Comment:
This gap is shortened by 2 slots. 
One slot allows for the new `xDomainMsgSender` value to be stored in the base `CrossDomainMessenger` contract before the gap.
The other slot makes space at the end of the gap for the `_initialized` slot.